index.html

```diff
@@ -1,192 +1,308 @@
-        
-        <p><strong>Shane’s story is a prime example of why funding for cancer research is crucial!</strong></p>
-        
-        <p>Don’t forget to protect your skin from the sun and get your skin checked by a dermatologist! For help with prevention and early detection visit <a href="https://www.aad.org/spot-skin-cancer" target="_blank">SPOT Skin Cancer™</a>.</p>
-        
-      </section>
-      
-      <section id="get-naked">
-        <h2>#GetNaked</h2>
-            <p>When’s the last time you got naked? Shane was a huge advocate for early detection through annual skin checks.</p>
-            <img src="/common/images/get-naked-long.png">
-      </section>
-      
-      <section id="help-the-cause">
-        <h2>Help the Cause</h2>
-        
-        <article id="miles-for-melanoma">
-          <h3>Miles for Melanoma Walk</h3>
-          <p>Join <a href="http://join.melanoma.org/site/TR/5kRunWalk/MilesforMelanoma?team_id=2300&pg=team&fr_id=1170" target="_blank">Shane's Miles for Melanoma Team</a> and/or donate to the cause!</p>
-          
-          <p>We'll all be participating in a 5k run/walk at Montrose Harbor on July 10th, 2016.</p>
-        </article>
-        
-        <article id="dayna-marathon">
-          <h3>2016 Chicago Marathon</h3>
-          <div class="row">
-            <div class="col-xs-12 col-md-6">  
-              <p>Join or donate to <a href="http://main.acsevents.org/site/TR?px=31685749&fr_id=77063&pg=personal" target="_blank">Dayna’s Chicago Marathon team</a> supporting the American cancer Society. She'll be running October 9th, 2016!</p>
-              
-              <img id="dayna" src="/common/images/dayna.jpg" alt="Dayna and Shane">
+<head>
+	<meta charset="UTF-8">
```

Comment:
@paulstanleyjohnson why the 4 space indentions? 2 is standard for HTML - although it really comes down to preference - 2 is typically better because elements these days tend to get nested even further so using 4 has great potential to run some markup off the screen or wrap it, when we can avoid it by using the widely-used 2 spaces.